### ISSUE TITLE
Attempting to boot Debian 12 LXDE results in a black screen.

### ISSUE THREAD

**Issue by funkymokey** [ading2210/shimboot#197]
**Describe the bug**
When attempting to boot Debian 12 LXDE, for a moment it will show "selected /dev/sda4 moving mounts to newroot  switching root", then go black.
**To Reproduce**
1. Boot hana shimboot shim
2. Click on the 5th option (multiple chrome os roots appear)
3. See "selected /dev/sda4 moving mounts to newroot  switching root"

**Expected behavior**
I expected to boot into a Debian 12 LXDE environment.
**Screenshots / Photos**
I can't add any photos because the screen reflects my face, and nobody wants to see me. But it looks exactly like the shimboot loader. But with 4 chrome os root options, and pressing 5 (Debian) shows "selected /dev/sda4 moving mounts to newroot switching root", and then goes black.
**Target Chrome OS Device (please complete the following information):**
 - Board Name (e.g. hana)
 - Device Name (e.g. hana)
 - Shimboot version (e.g. v1.2.1)

**Build Device (please complete the following information):**
 - OS: [Debian 12]
 - Neofetch out [jordan@debian
OS: Debian GNU/Linux 12 (bookworm) x86_64
Host: HP Laptop 15-dw1xxx
Kernel: 6.1.0-28-amd64
Uptime: 12 hours, 27 mins
Packages: 1834 (dpkg)
Shell: bash 5.2.15
Resolution: 1366x768
DE: GNOME 43.9
WM: Mutter
WM Theme: Adwaita
Theme: Adwaita [GTK2/3]
Icons: Adwaita [GTK2/3]
Terminal: gnome-terminal
CPU: Intel Pentium 6405U (4) @ 2.400GHz
GPU: Intel Comet Lake-U GT2 [UHD Graphics 620]
Memory: 1744MiB / 3720MiB]

**Additional context**
I know it is NOT the SD card's fault, since a USB drive doesn't work either, which is what I used to get Debian 12 on octopus boards. 
I have tried KDE and XFCE as well. Only recieving the same result.
(I completely deleted the old issue I had created because I didn't like how sloppy and unfinished it was.)

**Comment by Fighterhen63** [comment]
Can you boot into ChromeOS sucessfully?

**Comment by funkymokey** [comment]
> Can you boot into ChromeOS sucessfully?

No. I had a chat with somebody, and we determined that Gnome, and only Gnome works on a hana board. This could easily be a false alarm or maybe something wrong with shimboot. 
(using tasksel in the gnome terminal using root downloads the packages for a different desktop, but attempting to boot the Debian partition results in a black screen of nothing.)

**Comment by Wigaroo** [comment]
Is it missing permisions?
